spec/requests/racers_spec.rb

```diff
@@ -0,0 +1,111 @@
+require 'rails_helper'
+
+NUMBER_OF_RACERS = 3
+
+RSpec.describe 'Racers', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/racers/', headers: { 'Accept': 'application/json' }
+      expect(response).to have_http_status(:success)
+    end
+
+    context 'when there are no records' do
+      it 'returns an empty array' do
+        get '/racers/', headers: { 'Accept': 'application/json' }
+        expect(response.body).to eq('[]')
+      end
+    end
+
+    context 'when there are saved records' do
+      let!(:racers) { create_list(:racer, NUMBER_OF_RACERS) }
+      it 'returns an array of racers' do
+        get '/racers/', headers: { 'Accept': 'application/json' }
+        expect(JSON.parse(response.body).size).to eq(NUMBER_OF_RACERS)
```

Comment:
You don't need this constant. You can use `3` hardcoded here ou you as as `let`. Remember that a test also works as documentation.